packages/terra-date-input/src/DateInput.jsx

```diff
@@ -654,20 +661,33 @@ class DateInput extends React.Component {
       dateValue = `${year}-${month}-${day}`;
     }
 
+    let format = `(${this.props.intl.formatMessage({ id: 'Terra.date.input.dayPlaceholder' })} ${this.props.intl.formatMessage({ id: 'Terra.date.input.monthLabel' })} ${this.props.intl.formatMessage({ id: 'Terra.date.input.yearPlaceholder' })})`;
+
+    if (DateInputUtil.computedDisplayFormat(this.props.displayFormat, this.props.intl.locale) === 'month-day-year') {
+      format = `(${this.props.intl.formatMessage({ id: 'Terra.date.input.monthLabel' })} ${this.props.intl.formatMessage({ id: 'Terra.date.input.dayPlaceholder' })} ${this.props.intl.formatMessage({ id: 'Terra.date.input.yearPlaceholder' })})`;
```

Comment:
nit: Should we rename the translation keys (`Terra.date.input.yearPlaceholder`; `Terra.date.input.dayPlaceholder`, etc) to something like `yearLabel`/`dayLabel` since these are no longer used as placeholders?

Reply:
Updated the name of translation keys:
`dayPlaceholder` -> `dayFormatLabel`
`monthPlaceholder` -> `monthFormatLabel`
`yearPlaceholder` -> `yearFormatLabel`